### ISSUE TITLE
[Bug-gmx_MMPBSA_ana]: Core dumped when attempting to plot line graphs

### ISSUE THREAD

**Issue by wrmartin** [Valdes-Tresanco-MS/gmx_MMPBSA#408]
### Bug summary

I'm looking at the interaction between two domains of the same protein, split by the index file. The trajectory is 15,000 frames. I've loaded the decomp and energy for complex, receptor, and ligand. Attempting to plot any of the line plots results in an error.

### Terminal output

```bash
Traceback (most recent call last):
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/GMXMMPBSA/analyzer/customitem.py", line 371, in fn_btn_group
    self.plotting_line(checked)
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/GMXMMPBSA/analyzer/customitem.py", line 398, in plotting_line
    self.lp_subw = LineChart(line_plot_data, self.line_chart_action, options=options, item_parent=self)
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/GMXMMPBSA/analyzer/plots.py", line 215, in __init__
    self.line_plot_ax = sns.lineplot(data=self.data, color=rgb2rgbf(options[('Line Plot', 'line-color')]),
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/seaborn/_decorators.py", line 46, in inner_f
    return f(**kwargs)
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/seaborn/relational.py", line 710, in lineplot
    p.plot(ax, kwargs)
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/seaborn/relational.py", line 471, in plot
    for sub_vars, sub_data in self.iter_data(grouping_vars, from_comp_data=True):
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/seaborn/_core.py", line 983, in iter_data
    data = self.comp_data
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/seaborn/_core.py", line 1054, in comp_data
    with pd.option_context('mode.use_inf_as_null', True):
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/pandas/_config/config.py", line 441, in __enter__
    self.undo = [(pat, _get_option(pat, silent=True)) for pat, val in self.ops]
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/pandas/_config/config.py", line 441, in <listcomp>
    self.undo = [(pat, _get_option(pat, silent=True)) for pat, val in self.ops]
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/pandas/_config/config.py", line 135, in _get_option
    key = _get_single_key(pat, silent)
  File "/home0/martinw3/anaconda3/envs/gmxMMPBSA1.6/lib/python3.9/site-packages/pandas/_config/config.py", line 121, in _get_single_key
    raise OptionError(f"No such keys(s): {repr(pat)}")
pandas._config.config.OptionError: "No such keys(s): 'mode.use_inf_as_null'"
Aborted (core dumped)
```

### Actual outcome

gmx_MMPBSA_ana exits.

### Expected outcome

Display of the plot.

### Operating system

CentOS 7

### gmx_MMPBSA_ana Version

1.6.1

### Python version

3.9.16

### Installation

pip

**Comment by wrmartin** [comment]
Turns out this is a pandas version issue. the env.yml file for version 1.6.1 has pandas>=1.2.2, while the dev page is pandas==1.2.2. The current version of pandas is 2.0.3, and downgrading fixes this issue.

**Comment by Valdes-Tresanco-MS** [comment]
Sorry for that. We need to generate the v1.6.2 package to solve these problems. We will do it as soon as possible.

**Comment by Valdes-Tresanco-MS** [comment]
fixed in release v1.6.2
